src/application-insights/azext_applicationinsights/_help.py

```diff
@@ -70,10 +70,22 @@
     examples:
       - name: Connect AI to a web app and enable both profiler and snapshot debugger for the web app.
         text: |
-          az monitor app-insights component connect-webapp -g myRG -n myApp --enable-profiler --enable-snapshot-debugger
+          az monitor app-insights component connect-webapp -g myRG -a myAI --web-app myApp --enable-profiler --enable-snapshot-debugger
```

Comment:
`-a` ?

Reply:
c.argument('application', options_list=['--app', '-a'], id_part='name', help='GUID, app name, or fully-qualified Azure resource name of Application Insights component. The application GUID may be acquired from the API Access menu item on any Application Insights resource in the Azure portal. If using an application name, please specify resource group.')